lib_test/unix/dune

```diff
@@ -0,0 +1,9 @@
+(executable
+  (name      time_server)
+  (libraries dns-lwt-unix))
+
+(alias
+  (name   runtest)
+  (deps   time_server.exe)
+;  (action (run %{deps} --help))
```

Comment:
If this is supposed to be commented-out maybe we should add a comment explaining why.